isites_migration/utils.py

```diff
@@ -82,6 +82,20 @@ def export_files(keyword):
                 file_path = os.path.join(root, file)
                 z_file.write(file_path, file_path[zip_path_index:])
         z_file.close()
+
+        # we want to log the size of the data we exporting per TLT-2099
+        logger.info('Creating zip file %s' % zip_filename)
```

Comment:
sure, I think I was thinking the zip file had to be created first. But it makes sense that each file has it's own info as it's added to the archive.